tests/test_rspq.c

```diff
@@ -685,3 +698,49 @@ void test_rspq_highpri_overlay(TestContext *ctx)
     ASSERT_EQUAL_UNSIGNED(actual_sum[1], 123, "highpri sum is not correct");
     TEST_RSPQ_EPILOG(0, rspq_timeout);
 }
+
+void test_rspq_big_command(TestContext *ctx)
+{
+    TEST_RSPQ_PROLOG();
+    test_ovl_init();
+    DEFER(test_ovl_close());
+
+    uint32_t values[32];
+    for (uint32_t i = 0; i < 32; i++)
+    {
+        values[i] = RANDN(0xFFFFFFFF);
+    }
+    
+
+    uint32_t output[32] __attribute__((aligned(16)));
+    data_cache_hit_writeback_invalidate(output, 128);
+
+    rspq_write_t wptr = rspq_write_begin(test_ovl_id, 0x8, 33);
+    rspq_write_arg(&wptr, 0);
+    for (uint32_t i = 0; i < 32; i++)
+    {
+        rspq_write_arg(&wptr, i | i << 8 | i << 16 | i << 24);
```

Comment:
Why do you prefer to do an xor rather than writing and reading the same arguments?
Also do we depend on the initial state being all zeros when rspq is inited?

Reply:
> Why do you prefer to do an xor rather than writing and reading the same arguments?
No specific reason, just simulating a "do something on RSP".
> Also do we depend on the initial state being all zeros when rspq is inited?

Reply:
> Not that I see?

I mean would it behave the same if the initial dram contents were not all 0s? I think we do that when we do `rspq_init` before the test but I was just trying to validate my thinking.

Reply:
Oh right, but `TEST_BIG` is in `.data` so it is guaranteed to be zero-initialized. It would be different if it was in `.bss`, as in that case it wouldn't be initialized.